docs/pages/parallel-async.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Parallel and asynchronous processing
+layout: default
+---
+
+Python has a good ecosystem of libraries for multiprocessing (threads and GPU
+parallelisation), as well as asynchronous processing. Here, we list those that
+we've found to be useful, particularly for research applications and previous
+ARC projects.
+
+🟠 tools in the following may be preferred over 🟢, if there are external
+reasons to use a specific interface or parallelisation scheme. Possibly due to
+the nature of the research problem, the high-performance computing resources
+available or simply due to pre-existing code using a library like [pandas].
+
+| Name              | Short description                                                                                                                                                                                            | 🚦  |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :-: |
+| [multiprocess]    | A fork of [multiprocessing] which uses `dill` instead of `pickle` to allow serializing wider range of object types including nested / anonymous functions. We've found this rather more simple to work with. | 🟢  |
+| [multiprocessing] | The standard library module for distributing tasks across multiple processes                                                                                                                                 | 🟠  |
+| [Cython]          | Has [support for OpenMP based parallelism](https://cython.readthedocs.io/en/latest/src/userguide/parallelism.html)                                                                                           | 🟠  |
+| [mpi4py]          | support for MPI based parallelism                                                                                                                                                                            | 🟠  |
```

Comment:
```suggestion
| [Cython]          | Has [support for OpenMP based parallelism](https://cython.readthedocs.io/en/latest/src/userguide/parallelism.html)                                                                                           | 🟠  |
| [mpi4py]          | Support for _message passing interface_ (MPI) based parallelism                                                                                                                                              | 🟠  |
```
Changing case to be consistent with other rows + think its worth expanding MPI acronym as might not be familiar to some